third-party-reporter/test/index.spec.js

```diff
@@ -0,0 +1,7 @@
+const { equal } = require("assert");
+
+describe('Third party reporter usage suite', () => {
+  it('should equal true', () => {
+    equal(true, true);
```

Comment:
Consider creating "test/unit/" and putting this there.
Create "test/e2e/" and create a spec that actually runs the reporter.

Reply:
Have had a go at this. If you can see a cleaner way to do pls let me know.